src/components/MTableEditRow/index.js

```diff
@@ -147,10 +147,7 @@ function MTableEditRow(props) {
 
   function renderActions() {
     if (props.mode === 'bulk') {
-      if (props.detailPanel && !props.options.showDetailPanelIcon) {
-        return;
-      }
-      return <TableCell padding="none" key="key-actions-column" />;
+      return;
```

Comment:
The old class version had this fix but it wasn't carried over to the hook version. This is safe to keep.
https://github.com/material-table-core/core/blob/master/src/components/MTableEditRow/m-table-edit-row.js#L186

Reply:
Row based editing needs the action icons. Only bulk editing should hide all icons.

Reply:
In the code above, that means we should keep lines 150-152?

Reply:
150-152 should be removed, and it should return undefined as per the PR.  The class version `m-table-edit-row.js` does that.

Reply:
I'm not so sure about that. I remember adding that because there was an issue with bulk edit and detail panel. I'll have to dig some more to find a reference to the issue but I remember having to add that for a reason.

I just need to find some time to test it. I thought it was you that worked on the other issue with me but it doesn't look like it.

Reply:
It may have been needed in the past when the header row left a residual space. That's been removed in bulk edit mode, so the blank space isn't necessary